### PR TITLE
Improve infinite scrolling

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
Add infinite scrolling to local and federated timeline pages

Add infinite scrolling to an account page as well

Add e2e tests for the infinite scrolling of all the above + the existing explore page

### CODE REVIEW

**Review by CyberFlameGO — APPROVED** [review]
Looks like it works!

File: ui-e2e-tests/infinite-scrolling.spec.ts
Comment (dario-piotrowicz) [inline ui-e2e-tests/infinite-scrolling.spec.ts:84]: This should instead use the utilities defined in https://github.com/cloudflare/wildebeest/pull/285

We should merge that PR first and update the code here afterwards before merging

**Comment by dario-piotrowicz** [comment]
I've converted the PR to draft just to make sure we don't merge it before https://github.com/cloudflare/wildebeest/pull/285

File: ui-e2e-tests/infinite-scrolling.spec.ts
Comment (dario-piotrowicz) [reply ui-e2e-tests/infinite-scrolling.spec.ts:81]: For some reason playwright breaks when I try to import those utilities... :disappointed: 

I've at least updated this code to also not include real data, hopefully it should be good enough

**Review by xtuc — APPROVED** [review]

**Pull request merged**
